### PR TITLE
Updated fr.js

### PR DESCRIPTION
**Pull request by enygmate**
Keeping up with the recent changes

### CODE REVIEW

File: fr.js [hunk 1]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.market.stats.rare Rarest Items
-social.market.stats.circulation In Circulation
-social.market.stats.units units Worldwide
-social.trades.in Incoming
-social.trades.in.head Incoming Offers ({0})
-social.trades.out Outgoing
-social.trades.out.head Outgoing Offers ({0})
-social.trades.history History
-social.trades.history.head Trade History ({0})
-social.trades.decline Decline All
```
Comment (jackshallattack) [inline fr.js:773]: Refuser Tous

File: fr.js [hunk 2]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.profile.mods.none No published Mods from this user
-social.search Search Results
-social.market.stats.rare Rarest Items
-social.market.stats.circulation In Circulation
-social.market.stats.units units Worldwide
-social.trades.in Incoming
-social.trades.in.head Incoming Offers ({0})
-social.trades.out Outgoing
-social.trades.out.head Outgoing Offers ({0})
-social.trades.history History
```
Comment (jackshallattack) [inline fr.js:771]: Historique

File: fr.js [hunk 3]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-item.view.more More Information
-item.sell.cant Item cant be sold.
-item.sell.quick Are you sure you want to Quick Sell {0}?
-social.profile.stats Overall Stats
-social.profile.class Class Stats
-social.profile.sales Sales
-social.profile.sales.market Market Sales
-social.profile.skins Skins [{0}]
-social.profile.skins.none No published Skins from this user
-social.profile.games Games [{0}]
```
Comment (jackshallattack) [inline fr.js:757]: Jeux {0}

File: fr.js [hunk 4]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
 windows.playerlist.unflag Êtes-vous sûr de vouloir détaguer {0}?
 windows.playerlist.punish Êtes-vous sûr de vouloir prendre action sur {0}?
-item.view.open Open Model Viewer
-item.view.more More Information
-item.sell.cant Item cant be sold.
-item.sell.quick Are you sure you want to Quick Sell {0}?
-social.profile.stats Overall Stats
-social.profile.class Class Stats
-social.profile.sales Sales
-social.profile.sales.market Market Sales
```
Comment (jackshallattack) [inline fr.js:754]: Vente De Boutiques

File: fr.js [hunk 5]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
 windows.playerlist.flag Êtes-vous sûr de vouloir taguer {0}?
 windows.playerlist.unflag Êtes-vous sûr de vouloir détaguer {0}?
 windows.playerlist.punish Êtes-vous sûr de vouloir prendre action sur {0}?
-item.view.open Open Model Viewer
-item.view.more More Information
-item.sell.cant Item cant be sold.
-item.sell.quick Are you sure you want to Quick Sell {0}?
-social.profile.stats Overall Stats
-social.profile.class Class Stats
-social.profile.sales Sales
```
Comment (jackshallattack) [inline fr.js:753]: Vente

File: fr.js [hunk 6]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
 windows.playerlist.flag Êtes-vous sûr de vouloir taguer {0}?
 windows.playerlist.unflag Êtes-vous sûr de vouloir détaguer {0}?
 windows.playerlist.punish Êtes-vous sûr de vouloir prendre action sur {0}?
-item.view.open Open Model Viewer
-item.view.more More Information
-item.sell.cant Item cant be sold.
-item.sell.quick Are you sure you want to Quick Sell {0}?
-social.profile.stats Overall Stats
-social.profile.class Class Stats
```
Comment (jackshallattack) [inline fr.js:752]: Statistique De Classe

File: fr.js [hunk 7]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.profile.sales Sales
-social.profile.sales.market Market Sales
-social.profile.skins Skins [{0}]
-social.profile.skins.none No published Skins from this user
```
Comment (jackshallattack) [inline fr.js:756]: Pas De Costume Publier Par Ce Jouer

File: fr.js [hunk 8]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.profile.games Games [{0}]
-social.profile.games.none No published Games from this user
```
Comment (jackshallattack) [inline fr.js:758]: Pas De Jeux Publier Ce Ce Jouer

File: fr.js [hunk 9]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.profile.assets Assets [{0}]
-social.profile.assets.none No published Assets from this user
-social.profile.mods Mods [{0}]
-social.profile.mods.none No published Mods from this user
-social.search Search Results
-social.market.stats.rare Rarest Items
-social.market.stats.circulation In Circulation
-social.market.stats.units units Worldwide
-social.trades.in Incoming
-social.trades.in.head Incoming Offers ({0})
```
Comment (jackshallattack) [inline fr.js:768]: Offre Entrant

File: fr.js [hunk 10]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.profile.games.none No published Games from this user
-social.profile.assets Assets [{0}]
-social.profile.assets.none No published Assets from this user
-social.profile.mods Mods [{0}]
-social.profile.mods.none No published Mods from this user
-social.search Search Results
-social.market.stats.rare Rarest Items
-social.market.stats.circulation In Circulation
-social.market.stats.units units Worldwide
-social.trades.in Incoming
```
Comment (jackshallattack) [inline fr.js:767]: Entrant

File: fr.js [hunk 11]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.profile.games Games [{0}]
-social.profile.games.none No published Games from this user
-social.profile.assets Assets [{0}]
-social.profile.assets.none No published Assets from this user
-social.profile.mods Mods [{0}]
-social.profile.mods.none No published Mods from this user
-social.search Search Results
-social.market.stats.rare Rarest Items
-social.market.stats.circulation In Circulation
-social.market.stats.units units Worldwide
```
Comment (jackshallattack) [inline fr.js:766]: Nombre Mondiale

File: fr.js [hunk 12]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.profile.skins.none No published Skins from this user
-social.profile.games Games [{0}]
-social.profile.games.none No published Games from this user
-social.profile.assets Assets [{0}]
-social.profile.assets.none No published Assets from this user
-social.profile.mods Mods [{0}]
-social.profile.mods.none No published Mods from this user
-social.search Search Results
-social.market.stats.rare Rarest Items
-social.market.stats.circulation In Circulation
```
Comment (jackshallattack) [inline fr.js:765]: En Circulation

File: fr.js [hunk 13]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.profile.skins Skins [{0}]
-social.profile.skins.none No published Skins from this user
-social.profile.games Games [{0}]
-social.profile.games.none No published Games from this user
-social.profile.assets Assets [{0}]
-social.profile.assets.none No published Assets from this user
-social.profile.mods Mods [{0}]
-social.profile.mods.none No published Mods from this user
-social.search Search Results
-social.market.stats.rare Rarest Items
```
Comment (jackshallattack) [inline fr.js:764]: Items Le Plus Rares

File: fr.js [hunk 14]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.profile.sales.market Market Sales
-social.profile.skins Skins [{0}]
-social.profile.skins.none No published Skins from this user
-social.profile.games Games [{0}]
-social.profile.games.none No published Games from this user
-social.profile.assets Assets [{0}]
-social.profile.assets.none No published Assets from this user
-social.profile.mods Mods [{0}]
-social.profile.mods.none No published Mods from this user
-social.search Search Results
```
Comment (jackshallattack) [inline fr.js:763]: Résultats De Recherches

File: fr.js [hunk 15]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.market.stats.rare Rarest Items
-social.market.stats.circulation In Circulation
-social.market.stats.units units Worldwide
-social.trades.in Incoming
-social.trades.in.head Incoming Offers ({0})
-social.trades.out Outgoing
-social.trades.out.head Outgoing Offers ({0})
-social.trades.history History
-social.trades.history.head Trade History ({0})
```
Comment (jackshallattack) [inline fr.js:772]: Historique D'Échange

File: fr.js [hunk 16]
```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
...
-social.trades.decline Decline All
-social.trades.message ENABLE TRADE MESSAGES IN SETTINGS
-social.trades.warning You get no items from this trade!
```
Comment (jackshallattack) [inline fr.js:775]: Tu Reçois Pas De Objects De Cette Échange

**Review by jackshallattack — CHANGES_REQUESTED** [review]

File: fr.js (hunk 1 above)
Comment (enygmate) [reply fr.js:773]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 2 above)
Comment (enygmate) [reply fr.js:771]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 3 above)
Comment (enygmate) [reply fr.js:757]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 4 above)
Comment (enygmate) [reply fr.js:754]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 5 above)
Comment (enygmate) [reply fr.js:753]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 6 above)
Comment (enygmate) [reply fr.js:752]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 7 above)
Comment (enygmate) [reply fr.js:756]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 8 above)
Comment (enygmate) [reply fr.js:758]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 9 above)
Comment (enygmate) [reply fr.js:768]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 10 above)
Comment (enygmate) [reply fr.js:767]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 11 above)
Comment (enygmate) [reply fr.js:766]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 12 above)
Comment (enygmate) [reply fr.js:765]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 13 above)
Comment (enygmate) [reply fr.js:764]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 14 above)
Comment (enygmate) [reply fr.js:763]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 15 above)
Comment (enygmate) [reply fr.js:772]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

File: fr.js (hunk 16 above)
Comment (enygmate) [reply fr.js:775]: Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections

**Review by Sayrix — APPROVED** [review]
Bonne traduction.

**Comment by R3dlessX** [comment]
Good translate

**Review by Christian-Martins — APPROVED** [review]
Perfect!

**Pull request merged**